zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementFacade.kt

```diff
@@ -218,6 +235,7 @@ class ObjectManagementFacade(
         accessObject: ObjectManagementAccessObject,
         objectName: String,
         searchString: String?,
+        ordering: String = "",
```

Comment:
I don't see this being used anywhere right now. Should this be used in the methods that make use of this private method?